File: 릿코드/1-two-sum/1-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] index = new int[2];
        for(int i=0, len=nums.length; i<len; i++) {
            for(int j=i+1; j<len; j++) {
                if(nums[i] + nums[j] == target) {
                    index[0] = i;
                    index[1] = j;
                    return index;
                }
            }
        }
        return index;
    }
}

File: 릿코드/1-two-sum/1-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
const twoSum = (nums, target) => {
    for(let i=0, len=nums.length; i<len; i++) {
        for(let j=0; j<len; j++) {
            if(i === j) continue;
            if(nums[i] + nums[j] === target) return [i, j];
        }
    }
};